requirements-dev.in

```diff
@@ -1,18 +1,6 @@
 -c requirements.txt
```

Comment:
It's a philosophical feeling more than a practical one, but I feel like "packages a developer might want to have around" (like ipython) should be kept separate from "dependencies of this package" (in setup.cfg).

Reply:
You'd also still want this file around to contain this `-c` line, because as far as I can see, `pip-compile` doesn't let you provide that directly to the command line.

Reply:
Philosophical feelings are still relevant!

But you make valid practical points. Conceded.